grails-app/controllers/au/org/ala/spatial/service/LoginInterceptor.groovy

```diff
@@ -39,21 +39,40 @@ class LoginInterceptor {
             return true
         }
 
+        //Calculating the required permission.
+        def permissionLevel = null;
+        //Permission on method has the top priority
+        if ( method?.isAnnotationPresent(RequirePermission)) {
```

Comment:
whitespace is inconsistent - remove `( method` space

Reply:
Done

Reply:
reformatted

Reply:
reformatted